internal/constraints/constraints.go

```diff
@@ -113,7 +114,7 @@ func NewPrimeConditional(auth *authentication.Auth, pj projectable, subshellName
 		"Url":       pjURL,
 		"Commit":    pjCommit,
 		"Branch":    pjBranch,
-		"Path":      pjPath,
+		"Path":      pjDir,
```

Comment:
That was not my intent. From what I gather in https://github.com/ActiveState/cli/pull/2558/files#diff-0bc5f0643bb7ae3a8beadb795df406943b58af413f51b6e318c2ea83a63b89adL102-L104, "Path" expands to the directory of the project file. We already have a `project.Dir()` method, so I'm using it. I wanted to rename the variable to be more descriptive because it's a directory, not a path.

Reply:
After looking again; I missed that it was doing `filepath.Dir()` before on the pjPath, so this actually simplifies it. My bad!